.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+### Link to JIRA
+
+[PLAT-XYZ](https://dominodatalab.atlassian.net/browse)
```

Comment:
Python-domino is a public repo, but our JIRA is not open to public. So this could cause some confusion. Also, external collaborators can also come with some changes, and they obviously won't have a Domino JIRA open for their changes.

How should we address this problem?

Reply:
Going forward will we can create a Jira ticket and assign it to each issue to be resolved.